linked_list/linked_list.py

```diff
@@ -13,67 +12,157 @@ def __init__(self):
 
     # returns the value in the first node
     # returns None if the list is empty
-    # Time Complexity: ?
-    # Space Complexity: ?
+    # Time Complexity: O(1)
+    # Space Complexity: O(1)
     def get_first(self):
-        pass
-
+        if self.head is None:
+            return None
+        else: 
+            return self.head.value
 
     # method to add a new node with the specific data value in the linked list
     # insert the new node at the beginning of the linked list
-    # Time Complexity: ?
-    # Space Complexity: ?
+    # Time Complexity: O(1)
+    # Space Complexity: O(1)
     def add_first(self, value):
-        pass
-
+        new_node = Node(value)
+        new_node.next = self.head
+        self.head = new_node

```

Comment:
this works just fine! We can also exploit the default parameter in the `Node` class and add the next pointer all at once:

```suggestion
        self.head = Node(value, self.head)
```